pkg/bgpv1/api/conversions.go

```diff
@@ -16,85 +16,76 @@ import (
 	"github.com/cilium/cilium/pkg/bgpv1/types"
 )
 
-func ToAgentAfi(s string) types.Afi {
-	switch s {
-	case "ipv4":
-		return types.AfiIPv4
-	case "ipv6":
-		return types.AfiIPv6
-	case "l2vpn":
-		return types.AfiL2VPN
-	case "ls":
-		return types.AfiLS
-	case "opaque":
-		return types.AfiOpaque
+func ToAPIAfi(afi types.Afi) string {
```

Comment:
Why this adds back the code removed in https://github.com/cilium/cilium/pull/28035?

Reply:
Thanks for catching this. It must have slipped back in during a rebase.